internal/services/datadog/datadog_monitor_tag_rule_resource_test.go

```diff
@@ -161,33 +142,8 @@ resource "azurerm_datadog_monitor_tag_rule" "test" {
 `, r.template(data))
 }
 
-func (r TagRulesDatadogMonitorResource) requiresImport(data acceptance.TestData) string {
-	return fmt.Sprintf(`
-%s
-
-resource "azurerm_datadog_monitor_tag_rule" "import" {
-  datadog_monitor_id = azurerm_datadog_monitor_tag_rule.test.datadog_monitor_id
-  name               = azurerm_datadog_monitor_tag_rule.test.name
-  log {
-    subscription_log_enabled = true
-  }
-  metric {
-    filter {
-      name   = "Test"
-      value  = "Testing-Logs"
-      action = "Include"
-    }
-  }
-}
-`, r.basic(data))
-}
-
 func (r TagRulesDatadogMonitorResource) update(data acceptance.TestData) string {
 	return fmt.Sprintf(`
-provider "azurerm" {
-  features {}
-}
-
```

Comment:
these should be specified in each test, can we revert this?

Reply:
It has been defined in template func.

Reply:
`provider` blocks should not be defined in the template function and should be specified in each test - can we revert this?

Reply:
Currently, in main branch, provider block is defined in both template function and each test. So I assume this is a bug. So I removed provider block from template function otherwise test case would fail.